core/dev-tools/templates/simple/luigi-config.js

```diff
@@ -0,0 +1,19 @@
+Luigi.setConfig({
+  navigation: {
+      nodes: [{
+          pathSegment: 'home',
+          hideFromNav: true,
+          hideSideNav: true,
+          loadingIndicator: {
+            enabled: false
+          },
+          viewUrl: './microfrontend.html'

```

Comment:
Relative paths are not working as soon as you navigate to a deeper path.
```suggestion
          viewUrl: '/microfrontend.html'
```